pkg/service/export.go

```diff
@@ -43,6 +44,10 @@ func Export(clientProducer client.Producer, serviceNamespace, svcName string, st
 		},
 	}
 
+	if useClustersetIP {
+		mcsServiceExport.SetAnnotations(map[string]string{lhconstants.UseClustersetIP: "true"})
```

Comment:
```suggestion
		mcsServiceExport.SetAnnotations(map[string]string{lhconstants.UseClustersetIP: strconv.FormatBool(true)})
```

Reply:
This is the tricky one. Way flags work, default is false. But I don't want to set anything if user hasn't given an explicit option. In this case, setting it to `false` is different from not setting it at all. Not setting it means it will use the global setting on cluster. Setting false means even if global is true, it will not use clustersetip for this service.

I realize the way I've done means user can never set `false` explicitly, but I went with my option for backwards compatibility where existing usage is to not have it set at all.

Any suggestions on how we can handle it better?

Reply:
yeah make it a string and verify it's either "true" or "false".